packages/scaffolding-cli/src/domain/config/questions.ts

```diff
@@ -69,7 +69,7 @@ export function cliQuestions(default_project_name: string): Array<PromptQuestion
         {
             "type": "select",
             "name": "deployment",
-            "message": "Select Target Deployment",
+            "message": "Define target deployment",
```

Comment:
Maybe we rethink this. Could be `Define target strategy`? eg. `deployment pipeline`, `CI/CD` tool? 

@dnitsch @leobarnesamido should we think about `None` as an option too? i.e. just don't copy over the pipelines? It could be good for clients sites that have a pipeline strat in place?